src/components/ProductCard.js

```diff
@@ -4,16 +4,24 @@ import { Link } from 'react-router-dom';
 
 class ProductCard extends React.Component {
   render() {
-    const { props } = this.props;
-    const { title, price, id } = props;
+    const { product, addToCart, cartList } = this.props;
+    const { title, price, id, thumbnail } = product;
 
     return (
-      <div data-testid="product">
-        <div>{title}</div>
-        <div>{price}</div>
-        <Link to={ { pathname: `productDetails/${id}`, state: props } }>
+      <div className="productCard" data-testid="product">
+        <div className="product-card-item">{title}</div>
+        <div className="product-card-item">{price}</div>
+        <img src={ thumbnail } className="product-img" alt="product" />
+        <Link to={ { pathname: `productDetails/${id}`, state: { product, cartList } } }>
           <div data-testid="product-detail-link">View Details</div>
         </Link>
+        <button
+          type="button"
+          data-testid="product-add-to-cart"
+          onClick={ () => addToCart(product) }
+        >
+          add to card
```

Comment:
Acredito que tenha sido um typo

```suggestion
          add to cart
```

Reply:
Ok! Resolvido!